third_party/amd/lib/TritonAMDGPUToLLVM/TritonGPUToLLVM.cpp

```diff
@@ -275,5 +275,10 @@ std::unique_ptr<OperationPass<ModuleOp>> createConvertTritonAMDGPUToLLVMPass() {
   return std::make_unique<ConvertTritonAMDGPUToLLVM>(90);
 }
 
+std::unique_ptr<OperationPass<ModuleOp>>
+createConvertTritonAMDGPUToLLVMPass(int computeCapability) {
+  return std::make_unique<ConvertTritonAMDGPUToLLVM>(computeCapability);
```

Comment:
Is this `computeCapability` being used anywhere? Or is it just a place holder?

Reply:
It is used in `third_party/amd/lib/TritonAMDGPUToLLVM/ElementwiseOpToLLVM.cpp` to indicate whether it is MI300 or not (we used 300 to mean MI300 and 200 to represent MI200).

Reply:
I would agree that we need to converge on how we define and use computeCapability. As there are other places where it is used, I didn't change them as I didn't fully understand the consequences. So the original intention was to only change it as needed. With that said, if we think it is better to decide that now and clean up, that is also Okay. We will need to get consensus on how we define and use computeCapability and be consistent on that.

Reply:
Okay, you can put a TODO here maybe. I'll spend some time later to clean it upt.